02/app.js

```diff
@@ -1,2 +1,28 @@
-const x = 5;
-let result = 0;
\ No newline at end of file
+/*  const x = 5;
+    let result = 0;
+
+    for(let i=1; i<=x; i++) {
+        result = result + i;
+    }
+
+    console.log(result);
+
+    let i = 1;
```

Comment:
Jeśli korzystasz z tej samej zmiennej to powinieneś ją wyzerować tj. `result = 0` przed wykonaniem następnych operacji ;)